resources/quality/hms434/pla/hms434_0.8_pla_coarse.inst.cfg

```diff
@@ -0,0 +1,38 @@
+[general]
+version = 4
+name = Coarse
+definition = hms434
+
+[metadata]
+setting_version = 8
+type = quality
+quality_type = coarse
+
```

Comment:
Code style: remove empty space

Reply:
I don't think we really have any rule for this, do we? If it's more readable... Oh well, either way is fine with me.